letsencrypt/auth_handler.py

```diff
@@ -42,7 +42,7 @@ class AuthHandler(object):
         form of :class:`letsencrypt.achallenges.AnnotatedChallenge`
 
     """
-    def __init__(self, dv_auth, cont_auth, acme, account):
+    def __init__(self, dv_auth, cont_auth, acme, account, config=None):
```

Comment:
You should fix all tests. Config is typically passed first everywhere else in the code.